gpkit/nomials/variables.py

```diff
@@ -148,6 +148,8 @@ def __new__(cls, shape, *args, **descr):
             i = it.multi_index
             it.iternext()
             descr.update({"idx": i})
+            assert value_option is not None
+            # TODO refactor logic for value_option and remove above assert
```

Comment:
if the `if value_option in descr` never returns true, then `value_option` takes the value `sp_init`. I agree this is likely not working as intended. @bqpd needs to review to determine the intent. I took his fix to the pylint for now.